Frameworks/UIKit/UIView.mm

```diff
@@ -198,24 +199,68 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class],
-                                            [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],
-                                            [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class] };
```

Comment:
sounds like things we should have tests for. What do we need to be able to do this? Can we avoid fullblown UI automation via mocking?

---
In reply to: [78622666](https://github.com/Microsoft/WinObjC/pull/1002#discussion_r78622666) [](ancestors = 78622666)